roles/elasticsearch/tasks/elasticsearch-security.yml

```
@@ -185,20 +185,20 @@
   block:
     - name: Check the existance of cert on Ansible controler
       ansible.builtin.stat:
-        path: /tmp/{{ ansible_hostname }}.p12
+        path:  "{{ lookup('config', 'DEFAULT_LOCAL_TMP') | dirname }}/{{ ansible_hostname }}.p12"
       register: elasticsearch_check_temporary_cert
 
     - name: Move temporary cert on Ansible controler
       ansible.builtin.copy:
-        src: /tmp/{{ ansible_hostname }}.p12
-        dest: "/tmp/{{ ansible_hostname }}.p12_{{ ansible_date_time.iso8601_micro }}"
+        src: "{{ lookup('config', 'DEFAULT_LOCAL_TMP') | dirname }}/{{ ansible_hostname }}.p12"
+        dest:  "{{ lookup('config', 'DEFAULT_LOCAL_TMP') | dirname }}/{{ ansible_hostname }}.p12_{{ ansible_date_time.iso8601_micro }}"
         mode: preserve
       when: elasticsearch_check_temporary_cert.stat.exists
       register: elasticsearch_move_cert_file
 
     - name: Remove temporary cert on Ansible controler
       ansible.builtin.file:
-        path: /tmp/{{ ansible_hostname }}.p12
+        path:  "{{ lookup('config', 'DEFAULT_LOCAL_TMP') | dirname }}/{{ ansible_hostname }}.p12"
```

```suggestion
        path: "{{ lookup('config', 'DEFAULT_LOCAL_TMP') | dirname }}/{{ ansible_hostname }}.p12"
```
I think, I see a pattern. :-)

---

yes, somehow my search/replace vim regex seem to had a hiccup ... or it was rather pecak and not regex? Anyhow, thank you for you attention and suggestion.